boards/arm/arduino_nano_33_ble/arduino_nano_33_ble_defconfig

```diff
@@ -19,3 +20,8 @@ CONFIG_BOOTLOADER_BOSSA_LEGACY=y
 
 # additional board options
 CONFIG_GPIO_AS_PINRESET=y
+
+#USB ACM device for serial
+CONFIG_UART_INTERRUPT_DRIVEN=y
```

Comment:
That is not right, check out how nRF52840 dongle configures UART https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/arm/nrf52840dongle_nrf52840/Kconfig.defconfig#L36

Reply:
nRF52840 as i understand is enabling UART via USB CDC_ACM. What i am trying to achieve with this change is to enable serial console by connecting Arduino Nano 33 BLE to host via USB-TTL cable.